exercises/exercise_3/README.md

```diff
@@ -1,266 +1,255 @@
+    $ docker app install myrepo/myapp:mytag --name myinstallation --target-context=mycontext
+    $ docker app install bundle.json --name myinstallation --credential-set=mycredentials.yml
+
+    Options:
+          --credential-set stringArray    Use a YAML file containing a credential set or a credential set
+                                          present in the credential store
+          --insecure-registries strings   Use HTTP instead of HTTPS when pulling from/pushing to those registries
+          --kubernetes-namespace string   Kubernetes namespace to install into (default "default")
+          --name string                   Installation name (defaults to application name)
+          --orchestrator string           Orchestrator to install on (swarm, kubernetes)
+          --parameters-file stringArray   Override parameters file
+          --pull                          Pull the bundle
+      -s, --set stringArray               Override parameter value
+          --target-context string         Context on which the application is installed (default: <current-context>)
+          --with-registry-auth            Sends registry auth
+    ```
+    </details>
+
+    You'll see that we need to specify the name for our application and can set the `target-context`. Remember when we talked about Docker contexts? This will be where we use that. This allows us to run the commands in the dev instance, but have the deploy actually _happen_ on our Swarm cluster. Cool, huh?
+
+2. Let's deploy the application bundle using the `docker app install` command. Specify the name of your app as `voting-app`
+
+    <details>
+      <summary>Solution/Full Output</summary>
+
+      ```console
+      $ docker app install voting-app --target-context swarm
+      Creating network front-tier
+      Creating network back-tier
+      Creating service voting-app_vote
+      Creating service voting-app_redis
+      Creating service voting-app_db
+      Creating service voting-app_worker
+      Creating service voting-app_result
+      Application "voting-app" installed on context "swarm"
+      ```
+    </details>
+
+    :tada: The application stack is now deployed! Hooray!
+
+
+## Viewing Installed Applications
+
+Another handy tool is the `docker app ls` command, which allows us to see all currently installed applications.
+
+1. In your dev instance, run the `docker app ls` command. 
+
+    <details>
+      <summary>Full output</summary>
+
+      ```console
+      $ docker app ls
+      INSTALLATION APPLICATION LAST ACTION RESULT CREATED MODIFIED REFERENCE
+      ```
+    </details>
+
+    Huh... nothing showed up. Why?
+
+2. If we look at help for `docker app ls`, we'll see an option for `--target-context` again. When we first ran the `ls` command, it asked Docker for the app installations on our current machine, which has none. Go ahead and specify the `--target-context` and see if you get a better result.
```

Comment:
We can also specify this using the **`DOCKER_TARGET_CONTEXT`** env var. I guess it could be added as a **NOTE**?